documentation/templatetags/documentation_extras.py

```diff
@@ -8,12 +8,8 @@
 
 @register.filter
 def get_absolute_url_for_host(map, host):
```

Comment:
Once we verify that this is working I will just simply this to a method in the Map Model but for now, I wanted to stick with what we knew.

Reply:
sorry, thanks for explaining, I think that means you can ignore my other comment below.